cmd/talosctl/cmd/talos/config.go

```diff
@@ -127,8 +129,6 @@ var configContextCmd = &cobra.Command{
 			return err
 		}
 
-		c.Context = context
```

Comment:
isn't it what actually does the job in this command?

Reply:
It is now moved [here](https://github.com/siderolabs/talos/pull/6868/files#diff-79095d8dc4c63f2ca9526001f07b86e5f486c035b2832847c6c3925618f6ad02R60).

Reply:
I don't think this is right, no command should change the current context except for this one.

Reply:
`openConfigAndContext` receives empty context string in all cases except for `node`. Inside it verifies that `context` is empty and uses the one from the file. **Tim** wanted `talosctl config` to support `--context` flag, so we override Context field everywhere instead of only overriding it in `node`. That is actually quite consistent, no?

Reply:
Updating this field iirc changes the active context in talosconfig. Tim asked for the command to operate on the specified context, but not switch the context as a side effect. (I think that was it)

Reply:
Oh, I understand now - the change will result in override context being set as **active** one and saved in `talosconfig` when we didn't want that. Reverted and pushed the different set of changes.